sdk/postgresql/Azure.ResourceManager.PostgreSql/samples/Generated/Samples/Sample_PostgreSqlFlexibleServerBackupResource.cs

```diff
@@ -14,12 +14,73 @@ namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Samples
 {
     public partial class Sample_PostgreSqlFlexibleServerBackupResource
     {
+        // Create a new Backup for a flexible server
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
```

Comment:
Are we ignoring all tests?

Reply:
These are [samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#samples) they get generated as unit tests. 

Our tests are found in [PostgreSqlFlexibleServerTests.cs](https://github.com/Azure/azure-sdk-for-net/blob/54cbfebc9ba94be3897fa31207efc568f1a636d3/sdk/postgresql/Azure.ResourceManager.PostgreSql/tests/Scenario/PostgreSqlFlexibleServerTests.cs)